### DISCUSSION TITLE
Recovery Restic Repository

### DISCUSSION THREAD

**Discussion by hunterbit** [garethgeorge/backrest#545] [General]
Hi everyone,
the pc where Backrest is installed is no longer available due to a disk problem. I had to reinstall the operating system and install backrest but I don't know how to restore the old configuration.
The problem is that when I create a new REPO it always tries to run INIT but in reality the repository is already initialized by the old PC.
How can I connect the new configuration to an existing REPO?

Thanks.

**Comment by garethgeorge** [comment]
Hey, would you mind providing some logs / a screenshot of the error for this? 

Backrest is expected to skip calling `init` if it detects that the repo already exists https://github.com/garethgeorge/backrest/blob/main/pkg/restic/restic.go#L93-L130

**Reply by garethgeorge** [reply]
Edit: just tested this behavior again on my install to confirm and was able to re-import an existing repo `local-tank` as `local-tank2` using the same settings 
<img width="282" alt="image" src="https://github.com/user-attachments/assets/4c440516-5355-4ade-87d9-91b5027d386d">

**Reply by hunterbit** [reply]
I did another test and everything seems ok.
I must have done something wrong or there was a problem with the s3 server.
Thanks
